src/features/views/components/ViewBrowser/index.tsx

```diff
@@ -218,16 +218,7 @@ const ViewBrowser: FC<ViewBrowserProps> = ({
               apiRef={gridApiRef}
               autoHeight
               columns={colDefs}
-              components={{
-                Row: (props: GridRowProps) => {
-                  const item = props.row as ViewBrowserItem;
-                  return (
-                    <BrowserRow item={item} model={model} rowProps={props} />
-                  );
-                },
-              }}
-              disableSelectionOnClick
-              experimentalFeatures={{ newEditingApi: true }}
+              disableRowSelectionOnClick
               hideFooter
               isCellEditable={(params) => params.row.type != 'back'}
               onSortModelChange={(model) => setSortModel(model)}
```

Comment:
I believe the behavior of the sorting has changed here in the ViewBrowser. Previously it sorted alphabetically by default, now the default sorting is unclear, and when I go click a view and then go back to the list, the clicked view is suddenly at the bottom. Possibly the default GridSortModel has changed in v6?

Reply:
This sounds like #1456, which I believe @ziggabyte will be very familiar with. 😊

Reply:
Thank you for catching this! Need to implement a default sort method!

Reply:
This branch was created before this fix was merged from #1456 , so it should sort itself out when mergeing this to main!